### ISSUE TITLE
Browser back button not working

### ISSUE THREAD

**Issue by AInteriorB** [nextcloud/server#15265]
### Steps to reproduce
1. navigate into directory
2. use browser's back button

### Expected behaviour
going back

### Actual behaviour
nothing happens

### Server configuration

**Operating system**: Debian 10
**Web server:** Apache/2.4.38
**Database:** MariaDB 10.3.14
**PHP version:** PHP 7.3.4
**Nextcloud version:** 16.0.0
**Updated from an older Nextcloud/ownCloud or fresh install:** update
**Where did you install Nextcloud from:** web updater

**Signing status:**
<details>
<summary>Signing status</summary>
No errors have been found.
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
Enabled:
  - accessibility: 1.2.0
  - activity: 2.9.1
  - audioplayer: 2.7.0
  - cloud_federation_api: 0.2.0
  - comments: 1.6.0
  - dav: 1.9.2
  - federatedfilesharing: 1.6.0
  - files: 1.11.0
  - files_pdfviewer: 1.5.0
  - files_rightclick: 0.13.0
  - files_sharing: 1.8.0
  - files_texteditor: 2.8.0
  - files_trashbin: 1.6.0
  - files_versions: 1.9.0
  - files_videoplayer: 1.5.0
  - gallery: 18.3.0
  - logreader: 2.1.0
  - lookup_server_connector: 1.4.0
  - nextcloud_announcements: 1.5.0
  - notifications: 2.4.1
  - oauth2: 1.4.2
  - password_policy: 1.6.0
  - privacy: 1.0.0
  - provisioning_api: 1.6.0
  - recommendations: 0.4.0
  - serverinfo: 1.6.0
  - sharebymail: 1.6.0
  - support: 1.0.0
  - survey_client: 1.4.0
  - systemtags: 1.6.0
  - theming: 1.7.0
  - twofactor_backupcodes: 1.5.0
  - updatenotification: 1.6.0
  - viewer: 1.0.0
  - workflowengine: 1.6.0
Disabled:
  - admin_audit
  - encryption
  - federation
  - files_external
  - firstrunwizard
  - user_ldap
</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** yes

**Are you using an external user-backend, if yes which one:** no

**Comment by acs-ferreira** [comment]
Confirmed with Firefox@latest.

**Comment by skjnldsv** [comment]
@wiswedel please don't assign people, mention them and let them see if they have the time/knowledge to fix it.

**Comment by wiswedel** [comment]
@skjnldsv Roger that. Sorry.

**Comment by pawohl** [comment]
What I see in https://cloud.XXX.XXX/js/core/merged-template-prepend.js?v=XXXXXXX-2 is

```javascript
$(document).ready(initCore);

/**
// fallback to hashchange when no history support
if (window.history.pushState) {
	window.onpopstate = _.bind(OC.Util.History._onPopState, OC.Util.History);
}
else {
	$(window).on('hashchange', _.bind(OC.Util.History._onPopState, OC.Util.History));
}

/**
 * Get a variable by name
 * @param {string} name
 * @return {*}
 */
OC.get=function(name) {
```
Looks like someone (or a faulty tool) commented the change listener?

**Comment by krcroft** [comment]
Also confirmed in Chromium 74.0.3729.169 (Official Build) Built on Ubuntu, running on Ubuntu 19.04 (64-bit)

**Comment by MorrisJobke** [comment]
Fixed in #15640
